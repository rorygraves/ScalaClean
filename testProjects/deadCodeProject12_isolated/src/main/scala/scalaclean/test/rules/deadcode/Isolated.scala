package pkg

object Entry extends App

object I1 {
  println(I2)
}

object I2 {
  println(I1)
}
