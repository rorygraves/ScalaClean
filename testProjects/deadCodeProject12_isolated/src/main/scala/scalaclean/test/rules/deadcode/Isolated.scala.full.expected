package pkg

object Entry extends App
